manage_breast_screening/notifications/models.py

```diff
@@ -4,20 +4,39 @@
 
 from ..core.models import BaseModel
 
-BATCH_STATUSES = [
-    ("unscheduled", "Unscheduled"),
-    ("scheduled", "Scheduled"),
-    ("sent", "Sent"),
-    ("failed", "Failed"),
-]
-
-MESSAGE_STATUSES = [
-    ("pending_enrichment", "Pending enrichment"),
-    ("enriched", "Enriched"),
-    ("sending", "Sending"),
-    ("delivered", "Delivered"),
-    ("failed", "Failed"),
-]
+
+class BatchStatusChoices(models.Choices):
```

Comment:
Minor suggestion as I know it was called Batch_Statuses but maybe should be `MessageBatchStatusChoices` ?